src/pages/downloads/macos.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+import DownloadsPage from '.';
+import { OsType } from '../../data/downloads';
+
+export default function LinuxDownloads() {
```

Comment:
is everything a `LinuxDownload`?

Reply:
Linux is the default value https://github.com/jellyfin/jellyfin.org/pull/122/files#diff-91bb82f8d57d0f0b7c232e8df20f8997c93c56bb50c46197297937c06c325e6cR11